.github/workflows/github-actions.yml

```diff
@@ -0,0 +1,41 @@
+name: GitHub Actions
+on: [push]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version: ["3.8"]
```

Comment:
```suggestion
```

As you are only testing Netzob with one version of Python, it would be better to get rid of the “matrix” build strategy.

Reply:
The short term plan is to support Python 3.8 and Python 3.10, so that Netzob at least works on Ubuntu 20.04 and Ubuntu 22.04, thus the matrix strategy :)